### PR TITLE
Update dashHtmlComponents to 1.0.2 with support for async loading in R

### PR DESCRIPTION
**Pull request by rpkyle**
This PR proposes to update `dashHtmlComponents` to 1.0.2, while bundling in a vignette and many examples for the R online help.

This PR is related to, and required for plotly/dashR#157 to function properly.

### CODE REVIEW

File: dash-info.yaml [hunk 1]
```diff
@@ -8,3 +8,2473 @@ pkg_help_description: >
...
+              htmlDiv(list(
+                htmlBasefont(color="FF0000",
+                             face="Helvetica",
+                             size="+2"),
+                htmlP(children="If it works, this will be Helvetica but a couple point sizes larger.")
+                )
+               )
+            )
+
+            app$rqun_server()
```
Comment (Marc-Andre-Rivet) [inline dash-info.yaml]:
```suggestion
            app$run_server()
```

File: man/htmlBasefont.Rd [hunk 2]
```diff
@@ -66,3 +66,28 @@ those elements have the following types:
...
+  htmlDiv(list(
+    htmlBasefont(color="FF0000",
+                 face="Helvetica",
+                 size="+2"),
+    htmlP(children="If it works, this will be Helvetica but a couple point sizes larger.")
+    )
+   )
+)
+
+app$rqun_server()
```
Comment (Marc-Andre-Rivet) [inline man/htmlBasefont.Rd]:
```suggestion
app$run_server()
```

File: man/htmlFieldset.Rd [hunk 3]
```diff
@@ -73,3 +73,30 @@ those elements have the following types:
...
+
+library(dash)
+library(dashHtmlComponents)
+library(dashCoreComponents)
+
+app <- Dash$new()
+
+app$layout(htmlDiv(list(
+  htmlFieldset(
+    children = list('Choose your favourite Dash HTML component',
```
Comment (Marc-Andre-Rivet) [inline man/htmlFieldset.Rd]: Can't say we do this systematically, but do seem (tend) to default to English-US spelling.
```suggestion
    children = list('Choose your favorite Dash HTML component',
```

File: man/htmlLegend.Rd [hunk 4]
```diff
@@ -66,3 +66,31 @@ those elements have the following types:
...
+library(dash)
+library(dashHtmlComponents)
+library(dashCoreComponents)
+
+app <- Dash$new()
+
+app$layout(htmlDiv(list(
+  htmlFieldset(
+    children = list(
+                    htmlLegend('Select your favourite component'),
```
Comment (Marc-Andre-Rivet) [inline man/htmlLegend.Rd]:
```suggestion
                    htmlLegend('Select your favorite component'),
```

File: dash-info.yaml [hunk 5]
```diff
@@ -8,3 +8,2473 @@ pkg_help_description: >
...
+
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFieldset(
+                children = list('Choose your favourite Dash HTML component',
```
Comment (Marc-Andre-Rivet) [inline dash-info.yaml]:
```suggestion
                children = list('Choose your favorite Dash HTML component',
```

File: dash-info.yaml [hunk 6]
```diff
@@ -8,3 +8,2473 @@ pkg_help_description: >
...
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFieldset(
+                children = list(
+                                htmlLegend('Select your favourite component'),
```
Comment (Marc-Andre-Rivet) [inline dash-info.yaml]:
```suggestion
                                htmlLegend('Select your favorite component'),
```

File: dash-info.yaml [hunk 7]
```diff
@@ -1,10 +1,2479 @@
-pkg_help_description: >
+pkg_help_description: >-
```
Comment (Marc-Andre-Rivet) [inline dash-info.yaml]: Is this file generated?

Comment (rpkyle) [reply dash-info.yaml]: @Marc-Andre-Rivet This file is user-authored, and not auto-generated.

**Pull request merged**